manifests/experiences/twitter/pipeline.js

```diff
@@ -182,7 +182,7 @@ async function allAdvertisers(fileManager) {
   return { headers, items }
 }
 
-async function selectTargetingTree(fileManager, parameter) {
+async function selectTargetingTree(fileManager, _, parameter) {
```

Comment:
These arguments are starting to get a bit confusing and error-prone. Maybe having a single object as argument would be better, then we can read only the properties that are of interest for the function.

Reply:
This is not related to this PR ? Maybe we should create an issue for this ?

Reply:
Well it wasn't really a problem before this PR, but yeah we don't have to fix it now